modules/ROOT/pages/database-administration/aliases/manage-aliases-composite-databases.adoc

```diff
@@ -24,7 +24,7 @@ CREATE DATABASE `perennial-flowers`;
 Available database aliases in composite databases can be seen using `SHOW ALIASES FOR DATABASE`.
 The name of the composite database a particular database alias is part of appears in the returned `composite` column.
 
-The required privileges are described in the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/dbms-administration/#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
+The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
```

Comment:
if the link no longer points to Cypher Manual, should the link text also be updated?
```suggestion
The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[Access control -> DBMS administration].
```